pkgs/applications/video/makemkv/default.nix

```diff
@@ -1,34 +1,48 @@
-{ stdenv, fetchurl
-, openssl, qt5, libGLU_combined, zlib, pkgconfig, libav
+{ stdenv, mkDerivation, fetchurl, autoPatchelfHook
+, openssl, qtbase, libGLU_combined, zlib, pkgconfig, libav
 }:
 
-stdenv.mkDerivation rec {
-  name = "makemkv-${ver}";
-  ver = "1.14.4";
-  builder = ./builder.sh;
+mkDerivation rec {
+  pname = "makemkv";
+  version = "1.14.4";
 
   # Using two URLs as the first one will break as soon as a new version is released
   src_bin = fetchurl {
```

Comment:
src_bin and src_oss should move into a let expression - they shouldn't be "polluting" the attributes of the derivation.